sdk/storage/azure-storage-file-share/azure/storage/fileshare/_lease.py

```diff
@@ -7,8 +7,8 @@
 
 import uuid
 
-from typing import (  # pylint: disable=unused-import
-    Union, Optional, Any, TypeVar, TYPE_CHECKING
+from typing import (
+    Union, Optional, Any, TYPE_CHECKING
 )
```

Comment:
Ehh, probably not worth an iteration to this PR if it's the only change :). Maybe next PR if you remember, otherwise call it future-proofing is also fine.

Reply:
Yes, this will fit in one line. I will go through all the files one more time to ensure this standard is being followed everywhere. Thanks for pointing it out :)

Reply:
@jalauzon-msft What do you think about this? Should this be one line or should we keep it as

```python
from typing import (
     A, B, C
    TYPE_CHECKING
)
```

Reply:
It might be worth it to keep as two or more lines because it's easier to add, remove, or modify package imports, but. I can also see why one line might be good as well. If this does end up being just one line, should `TYPE_CHECKING` also be alphabetized just like the rest?